Gemfile

```diff
@@ -1,10 +1,3 @@
 source 'http://rubygems.org'
 
 gemspec
-
-case version = ENV['GRAPE_VERSION'] || '~> 0.14.0'
-when 'HEAD'
-  gem 'grape', github: 'ruby-grape/grape'
-else
-  gem 'grape', version
-end
```

Comment:
Removing this caused Travis to not build different versions!

https://github.com/ruby-grape/grape-swagger/blob/master/.travis.yml